Gems/AudioSystem/Code/Source/Editor/AudioControlsWriter.cpp

```diff
@@ -80,6 +79,18 @@ namespace AudioControls
                 index = index.sibling(++i, 0);
             }
 
+            auto fileIO = AZ::IO::FileIOBase::GetInstance();
+            AZStd::for_each(
+                m_foundLibraryPaths.begin(), m_foundLibraryPaths.end(),
+                [fileIO](AZStd::string& libraryPath) -> void
+                {
+                    AZStd::optional<AZ::u64> newLength = fileIO->ConvertToAlias(libraryPath.data(), libraryPath.size());
+                    if (newLength)
+                    {
+                        libraryPath.resize_no_construct(*newLength);
+                    }
+                    AZStd::to_lower(libraryPath.begin(), libraryPath.end());
```

Comment:
This could call the ConvertToAias function which returns an `AZStd::optional<AZ::IO::FixedMaxPath>`

If you need to modify the string contents of the path, you can always call `.Native()` on it which returns a **non-const** reference to the underlying string type.

Reply:
Will do.